README.md

```diff
@@ -1,2 +1,5 @@
 # openimis-fe-nepal_language_pack_js
 openIMIS translation to Nepali and english version of Nepal implementation.
+
+
+INSERT INTO tblLanguages ("LanguageCode", "LanguageName", "SortOrder") VALUES ('np', 'Nepali', 3);
```

Comment:
Why this here?

Reply:
I've updated the repository, you have to insert language information manually, so I left the code for convenience.

Reply:
@purusap can you confirm that you didn't had the language in tblLanguages?

Reply:
This step is mentioned in the link with instructions in the assembly module:
`in tblLanguages on database level add new language for example 'es' (Spanish language)`
So we can assume someone will need it.

Reply:
Once the mobile apps are migrated to modular FHIR, I suppose we won't need tblLanguages anymore. Will check with BS about this.

Reply:
> @purusap can you confirm that you didn't had the language in tblLanguages?

We do have "ne" Langauge in tblLanguages.